src/test/java/bio/terra/common/iam/AuthenticatedUserRequestTest.java

```diff
@@ -0,0 +1,75 @@
+package bio.terra.common.iam;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import bio.terra.common.exception.ApiException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.UUID;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Tag("unit")
+public class AuthenticatedUserRequestTest {
+
+  private static final Logger logger = LoggerFactory.getLogger(AuthenticatedUserRequestTest.class);
+
+  private static final String EmailAddress = "test@example.com";
+  private static final UUID RequestId = UUID.randomUUID();
+  private static final String SubjectId = "Subject";
+  private static final String Token = "0123.456-789AbCd";
+
+  public void validateJsonSerialization(AuthenticatedUserRequest request)
+      throws JsonProcessingException {
+    ObjectMapper objectMapper = new ObjectMapper();
+    String str = objectMapper.writeValueAsString(request);
```

Comment:
adding a test against deserializing hardcoded string is a good way to check for changes to backwards compatibility. While the test will be changed when the serialization format changes, it's a good reminder when the test breaks backwards compatibility and some assurance that it otherwise does not.

Reply:
I added a mechanism for easily adding hard-coded JSON test vectors.  If there's an easy way to grab some "real world" JSON examples from current stairway files, I would be happy to add those to the test vectors.

One thing to note is that it's also important that we're using the same Jackson `ObjectMapper` configuration as downstream serializers for this to be effective (there are some docs in the unit test on this).